code/XMLTester.xml

```diff
@@ -4058,7 +4058,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
 -->
-    <publisher profile_name="publisher_xml_conf_multicast_locators_profile">
+    <subscriber profile_name="publisher_xml_conf_multicast_locators_profile">
```

Comment:
```suggestion
    <subscriber profile_name="subscriber_xml_conf_multicast_locators_profile">
```

Reply:
If you decided to apply this suggestion, this profile name sure is used in the testing XML source code and you will change that too.